src/video_core/buffer_cache/buffer_cache.h

```diff
@@ -690,7 +690,10 @@ void BufferCache<P>::BindHostGraphicsUniformBuffer(size_t stage, u32 index, u32
     const VAddr cpu_addr = binding.cpu_addr;
     const u32 size = binding.size;
     Buffer& buffer = slot_buffers[binding.buffer_id];
-    if (size <= uniform_buffer_skip_cache_size && !buffer.IsRegionGpuModified(cpu_addr, size)) {
+    bool use_fast_buffer = size <= uniform_buffer_skip_cache_size &&
```

Comment:
```suggestion
   const bool use_fast_buffer = size <= uniform_buffer_skip_cache_size &&
```

For consistency with the surrounding code.